### ISSUE TITLE
TST test sample weight for NARX

### ISSUE THREAD

**Issue by MatthewSZhang** [scikit-learn-contrib/fastcan#67]
~~The sample weight of NARX cannot be checked by repeat or remove samples like scikit-learn does. Need specific test for it.~~

The sample weight only effect the loss function of narx. We can use static narx, which does not contain y and time delays, to test if the sample weight effect loss correctly by repeating or removing samples.
